Common/Sources/Extensions/HyperwalletFee.swift

```diff
@@ -36,7 +36,7 @@ public extension HyperwalletFee {
             description = FeeTypes.flat(flatFee).feeDescription()
         } else if let percentFee = percentFee {
             description = FeeTypes.percent(percentFee).feeDescription()
-        }
+        } 
```

Comment:
why space? Didn't this give swiftlint warning?

Reply:
Removed space.